test/libmpv_common.h

```diff
@@ -84,10 +84,58 @@ static inline void check_api_error(int status)
         fail("mpv API error: %s\n", mpv_error_string(status));
 }
 
+static void check_double(const char *property, double expect)
```

Comment:
```suggestion
MP_UNUSED static inline void check_double(const char *property, double expect)
```
instead of adding flag in meson, should work too.